src/components/manage/Blocks/Hero/schema.js

```diff
@@ -13,17 +13,25 @@ const messages = defineMessages({
     id: 'Link to',
     defaultMessage: 'Link to',
   },
+  Align: {
+    id: 'Alignment',
+    defaultMessage: 'Alignment',
+  },
 });
 
-const schemaHero = ({ intl }) => {
+const schemaHero = ({ intl, variation }) => {
   return {
     title: 'Block settings',
     required: [],
     fieldsets: [
       {
         id: 'default',
         title: intl.formatMessage(messages.more),
-        fields: ['linkTitle', 'linkHref'],
+        fields: [
+          'linkTitle',
+          'linkHref',
```

Comment:
Also I can't find where the `linkHref` is pointing to, at least it didn't worked as I tried.